### PR TITLE
Fix test failures

### PR DESCRIPTION
**Pull request by dinuish94**

### CODE REVIEW

File: docs/content/pom.xml [hunk 1]
```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+
+    <parent>
+        <groupId>org.wso2.ballerina-integrator</groupId>
+        <artifactId>ballerina-integrator</artifactId>
+        <version>1.0.0-alpha</version>
```
Comment (Maninda) [inline docs/content/pom.xml]:
```suggestion
        <version>1.0.0</version>
```

File: docs/pom.xml [hunk 2]
```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+
+    <parent>
+        <groupId>org.wso2.ballerina-integrator</groupId>
+        <artifactId>ballerina-integrator</artifactId>
+        <version>1.0.0-alpha</version>
```
Comment (Maninda) [inline docs/pom.xml]:
```suggestion
        <version>1.0.0</version>
```

File: vscode/snippet-generator/pom.xml [hunk 3]
```diff
@@ -28,7 +28,6 @@
     </parent>
 
     <modelVersion>4.0.0</modelVersion>
-    <groupId>org.wso2.ballerina-integrator</groupId>
     <artifactId>ballerina-integrator-snippet-generator</artifactId>
     <version>1.0.0-alpha</version>
```
Comment (Maninda) [inline vscode/snippet-generator/pom.xml]:
```suggestion
    <version>1.0.0</version>
```
And please search all the versions in the _pom.xml_ files and correct them.

File: docs/content/pom.xml.versionsBackup [hunk 4]
```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (IrushiL) [inline docs/content/pom.xml.versionsBackup]: Unwanted file?

File: docs/content/src/learn/integration-tutorials/sending-a-simple-message-to-a-service/testResults [hunk 5]
```diff
@@ -0,0 +1,16 @@
+Compiling source
```
Comment (IrushiL) [inline docs/content/src/learn/integration-tutorials/sending-a-simple-message-to-a-service/testResults]: Unwanted file?

File: docs/doc-generator/init.sh [hunk 6]
```diff
@@ -0,0 +1,14 @@
...
+JAR_FULL_NAME=$JAR_NAME-$VERSION-jar-with-dependencies.jar
+echo "Running jar file " $JAR_FULL_NAME
+java -jar target/$JAR_FULL_NAME > target/execution.log 2>&1
+
+if (grep -q "Invalid file path" target/execution.log)
+  then
+    echo "Invalid file path in INCLUDE_CODE tag. Mentioned file does not exists in the project. See more at execution.log"
+    exit 1
+  fi
+
```
Comment (IrushiL) [inline docs/doc-generator/init.sh]: New line?

File: docs/doc-generator/pom.xml.versionsBackup [hunk 7]
```diff
@@ -6,14 +6,12 @@
     <parent>
```
Comment (IrushiL) [inline docs/doc-generator/pom.xml.versionsBackup]: Unwanted file?

File: docs/pom.xml.versionsBackup [hunk 8]
```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (IrushiL) [inline docs/pom.xml.versionsBackup]: Unwanted file?

File: pom.xml.versionsBackup [hunk 9]
```diff
@@ -0,0 +1,223 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (IrushiL) [inline pom.xml.versionsBackup]: Unwanted file?

File: vscode/snippet-generator/pom.xml.versionsBackup [hunk 10]
```diff
@@ -0,0 +1,170 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (IrushiL) [inline vscode/snippet-generator/pom.xml.versionsBackup]: Unwanted file?

File: vscode/extension/pom.xml.versionsBackup [hunk 11]
```diff
@@ -0,0 +1,95 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (IrushiL) [inline vscode/extension/pom.xml.versionsBackup]: Unwanted file?

**Pull request merged**